.github/workflows/task_backend_tests.yml

```diff
@@ -32,6 +34,16 @@ jobs:
         with:
           repository: rotki/test-caching
           path: test-caching
+          fetch-depth: 0
+      - name: Ensure VCR branch is fully rebased
+        run: |
+          cd $HOME/work/rotki/rotki/test-caching
```

Comment:
here we could potentially replace that with working directory https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory but if it works as it is it's also ok